src/pybind/mgr/dashboard/frontend/src/app/shared/services/summary.service.ts

```diff
@@ -40,7 +42,46 @@ export class SummaryService {
     });
   }
 
+  /**
+   * Returns the current value of summaryData
+   *
```

Comment:
This comment won't bring any value, please remove it.

Reply:
This will inform the user that this will not create a new http request and simply return the current data.

Reply:
How about renaming the function to `getCurrentSummary` or `getCurrentValue` (`summaryService.getCurrentValue`) if the function name is not descriptive enough.

As this function doesn't do anything complicated and there isn't any unexpected behavior a comment shouldn't been necessary.

Reply:
I change the name of the method.
Documentation is always a nice to have, so I kept it.